assignments/objects.js

```diff
@@ -18,28 +18,66 @@ let example = {
 }
 
 // Write your intern objects here:
-
+let intern1 = {
+  "id": 1,
+  "email": "mmelloy0@psu.edu",
+  "name" : "Mitzi",
+  "gender" : "F",
+};
+let intern2=
+{
+  "id": 2,
+  "email": "kdiben1@tinypic.com",
+  "name" : "Kennan",
+  "gender" : "M",
+};
+let intern3=
+{
+  "id": 3,
+"email": "kmummery2@wikimedia.org",
+"name" : "Keven",
+"gender" : "M",
+};
+let intern4=
+
+{"id": 4,
+"email": "gmartinson3@illinois.edu",
+"name" : "Gannie",
+"gender" : "M",
+};
+let intern5=
+
+{"id": 5,
+"email": "adaine5@samsung.com",
+"name" : "Antonietta",
+"gender" : "F",
+};
 
 // ==== Challenge 2: Reading Object Data ==== 
 // Once your objects are created, log out the following requests from HR into the console:
 
 // Mitzi's name
-
+console.log (Object.entries(intern1)[2]);
```

Comment:
You can access the property directly with ```console.log(intern1.name)```. No need for all of the extra object properties like entries. Different strokes for different folks. I will continue to say that yours is absolutely not wrong but sometimes there are shorter ways to write it and programmers are lazy by nature.